client/sync_test.go

```diff
@@ -0,0 +1,117 @@
+// eventsgateway
+// +build unit
+// https://github.com/topfreegames/eventsgateway
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package client_test
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/topfreegames/eventsgateway/client"
+	pb "github.com/topfreegames/protos/eventsgateway/grpc/generated"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Sync Client", func() {
+	var (
+		c   *client.Client
+		now int64
+	)
+	name := "EventName"
+	props := map[string]string{
+		"prop1": "val1",
+		"prop2": "val2",
+	}
+
+	BeforeEach(func() {
+		var err error
+		c, err = client.NewClient("", config, logger, mockGRPCClient)
+		Expect(err).NotTo(HaveOccurred())
+		now = time.Now().UnixNano() / int64(time.Millisecond)
+	})
+
+	Describe("Send", func() {
+		var (
+			c   *client.Client
+			now int64
+		)
+		name := "EventName"
+		props := map[string]string{
+			"prop1": "val1",
+			"prop2": "val2",
+		}
+
+		BeforeEach(func() {
+			var err error
+			c, err = client.NewClient("", config, logger, mockGRPCClient)
+			Expect(err).NotTo(HaveOccurred())
+			now = time.Now().UnixNano() / int64(time.Millisecond)
```

Comment:
nit: `time.Now().UnixNano() / 1000000` instead?

Reply:
done

Reply:
@ghostec I think you forgot this one.

Reply:
yeap, changed in the other branch I'm working on, sorry :/